README.md

```diff
@@ -54,9 +54,7 @@ default to `docker`.
 
 ## Installation
 
-```
-$ cargo install cross --git https://github.com/cross-rs/cross
-```
+$ ```cargo install cross --git https://github.com/cross-rs/cross```
```

Comment:
this is technically not a valid code fence.

should be 
```
$ `cargo install cross --git https://github.com/cross-rs/cross`
```

however, I don't like that, instead I think


    ```sh
    cargo install cross --git https://github.com/cross-rs/cross`
    ```

would be good, not the added "`sh`"

Reply:
I like the latter: the `$` symbols are only useful when you need to show both input and output, and therefore the block isn't meant to be copied and pasted directly to a terminal.

Reply:
The dollar symbol is not needed, if click the copy button will get `$ cargo install ...`, Paste it to terminal directly and execute users will caught:
```
-bash: $: command not found
```
I think the `$` symbol is just a visual guidance says to user: Yes, this is a shell command.

So, let's make a quick decision, I prefer the first one you give, However, I can accommodate, Haha

Reply:
Lets go with 

	  ```sh
	  cargo install cross --git https://github.com/cross-rs/cross`
	  ```